src/util/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ if(GIT_FOUND)
       OUTPUT_STRIP_TRAILING_WHITESPACE
       WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
     )
-    configure_file(\${CUR}/version.cpp.in version.cpp)
+    configure_file(\${CUR}/version.cpp.in \${CBMC_SOURCE_DIR}/util/version.cpp)
```

Comment:
:no_entry_sign: Please drop this commit as this "fix" is not needed.

Reply:
without it version.cpp gets generated in the build dir. and the new version is ignored.

Reply:
The `version.cpp` in the build dir is compiled and linked into the build. Ignoring this file would result in a failed build as it would not exist elsewhere. The latest Ubuntu build on our release page shows the correct version number when I download and install it and it was built without this proposed change. I also end up with CBMC builds showing the correct version number for builds I produce locally, with altered version numbers in `config.inc`. Under what conditions have you seen incorrect version numbers?

If you have also built cbmc using `make` and the makefiles, then you can end up with a `version.cpp` in the `cbmc/src/util` directory. that would be a stale build artefact, from the build in place nature of the make build. It should not be committed to git. To my knowledge it is correctly ignored and neither overwritten or linked by the cmake build, if it exists.

Reply:
Sorry. You are right. My mistake.